solutions/gatekeeper-policies/naming-rules/README.md

```diff
@@ -0,0 +1,3 @@
+# GCP Gatekeeper Naming Policies
+
+This folder contains Gatekeeper naming policies for GCP resources.
```

Comment:
Similar to an earlier comment. We should add some documentation around the naming policies. This can be added in a smaller pr once this is merged.

Reply:
Sounds good. I did add specific information for the project naming policy here: solutions/gatekeeper-policies/naming-rules/project/README.md.

I would like to still see a table created here as well. :)

Reply:
I will create a new issue to track this work